### DISCUSSION TITLE
More scalable navigation experience

### DISCUSSION THREAD

**Discussion by pamelahu** [twilio-labs/paste#3333] [Office hours] [answered]
### Background

With the One Admin initiative, I need to rehouse more than 1100 Monkey pages into a new navigation experience. 

There's 5 levels of information that needs to be accounted for within One Admin:

![Screenshot 2023-07-18 at 5 17 13 PM](https://github.com/twilio-labs/paste/assets/18605252/3e7dff65-9941-497e-bee5-c6c426a00933)

In [these mocks](https://www.figma.com/file/UA7D8gwNwt8k7iSIRXvUIC/Old-%3C%3E-New-Monkey?type=design&node-id=467%3A129616&mode=design&t=wmvkE6prbRyQilk5-1) I walk through 2 ideas... Option 2 better adheres to how the new navigation UI works. 

### Feedback

Wondering if yall have any ideas on how to create a more scalable version of the navigation experience? We have a lot of technical constraints!! :(

### Relevant links

https://www.figma.com/file/UA7D8gwNwt8k7iSIRXvUIC/Old-%3C%3E-New-Monkey?type=design&node-id=467%3A129616&mode=design&t=wmvkE6prbRyQilk5-1

### Are you looking for visual design support?

- [ ] Yes

**Comment by serifluous** [comment]
Hey @pamelahu ! A few questions before we go into office hours:

1. Is the difference between Option 1 and 2 the switcher that's in the topbar in option 2?
2. Have you done any testing yet with the 2 options?
3. How tied are you and your team to having the "Account" and "Global" tabs in the sidebar?

**Comment by pamelahu** [comment]
1. That's a difference yes! Ultimately it's the hierarchy of visibly having One Admin as the top level (option 1) & so when a user clicks "Home" it brings them to the One Admin home page (where they can switch to a different center):
![Screenshot 2023-07-20 at 12 56 19 PM](https://github.com/twilio-labs/paste/assets/18605252/1f2fb25f-b0c9-4998-8080-529096a50881)

And option 2 as Customer Center as the top level so when you click home it goes to the Customer Center home... I guess there's no way to really get back to the home page but also you wouldn't really need to because you can access the different centers in the switcher. 

![Screenshot 2023-07-20 at 12 56 26 PM](https://github.com/twilio-labs/paste/assets/18605252/faa685b8-f0d1-4f4e-beef-1ff0f4129371)


3. No testing yet, doing calls next week

4. Open to other ideas!

@serifluous

**Comment by serifluous** [comment, marked as answer]
Thanks for coming to office hours! Here's the link to the recording https://twilio.zoom.us/rec/share/3teOI_Le1bUKBVDuBnyXhXGEg06pCJ1TCd05SYTqUktOsHRcrvFdjpxsk9tcvF-c.h-Y-1EHjWO_aJtU3

You can find the password in our slack channel.
